shared/src/industry.ts

```diff
@@ -9,6 +9,7 @@ export interface Industry {
   readonly additionalSearchTerms?: string;
   readonly defaultSectorId?: string;
   readonly roadmapSteps: AddOn[];
+  readonly nonEssentialQuestions: NonEssentialQuestion[];
```

Comment:
This is kinda confusing in that both Industry.ts and profileData.ts both have something called nonEssentialQuestions and one is a record while the other is an Array, could we consider more specific names here?

Reply:
thoughts on nonEssentialQuestionsArray and nonEssentialQuestionsRecord?

Reply:
or should I name the array to be nonEssentialQuestionMappings and leave the record as nonEssentialQuestions?

Reply:
I'm down either, the second seems a bit better to me

Reply:
@somabadri It's generally considering an antipattern to name something based on its type like `nonEssentialQuestionsArray` or `displayString` or something, because your IDE can tell you the type, so naming it for its type is redundant.

I'd recommend trying to name it based on the difference between them, domain-wise, like what is their purpose?

also, just be careful with the `nonEssentialQuestionMappings` idea because a `Map` is traditionally a synonym for a Record, so if anything is called Mappings, it should be a record, not an array